OptimizelySDK/ProjectConfig.cs

```diff
@@ -552,5 +559,32 @@ public Rollout GetRolloutFromId(string rolloutId)
             ErrorHandler.HandleError(new Exceptions.InvalidRolloutException("Provided rollout is not in datafile."));
             return new Rollout();
         }
+
+        /// <summary>
+        /// Get attribute ID for the provided attribute key
+        /// </summary>
+        /// <param name="attributeKey">Key of the Attribute</param>
+        /// <returns>Attribute ID corresponding to the provided attribute key. Attribute key if it is a reserved attribute</returns>
+        public string GetAttributeId(string attributeKey)
```

Comment:
Please add unit tests for this new module

Reply:
Already covered here:

Reply:
Oh, good. GitHub does not render large files by default and hence missed it.